### DISCUSSION TITLE
Annotations List - Html Support

### DISCUSSION THREAD

**Discussion by michaelvannevel** [grafana/grafana#37265] [Dashboards, Panels & Visualizations]
When the annotations panel was from ryanxtru(i believe it was this), I cloned his code and added html support.  This time around I'd like to go the proper route and request it.  Currently the annotations panel does not support html.  When you create a straight out grafana annotation you send in what, body, and tags.  Our body contains html.  This looks quite ugly in the annotations panel, and so I'm asking for html support to be added.

I was looking into how to do this, and found this article useful: https://stackoverflow.com/questions/39758136/render-html-string-as-real-html-in-a-react-component  I have never done react development, so i don't feel comfortable contributing directly.

Please let me know your thoughts

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
